lib/shared-sku-app/tsconfig.json

```diff
@@ -7,7 +7,11 @@
     "rootDir": "./src",
     "experimentalDecorators": true,
     "esModuleInterop": true,
-    "lib": ["dom", "es2015"],
+    "lib": [
+      "es6",
+      "dom",
+      "es2017"
+    ],
```

Comment:
What's with this change?

Reply:
tests were failing because TS compiler was complaining that `.includes` isn't a method on type of `string[]`